src/themes/default/pages/Category.vue

```diff
@@ -84,12 +84,13 @@ import config from 'config'
 import Columns from '../components/core/Columns.vue'
 import ButtonFull from 'theme/components/theme/ButtonFull.vue'
 import { mapGetters } from 'vuex'
+import uniq from 'lodash-es/uniq'
 import onBottomScroll from '@vue-storefront/core/mixins/onBottomScroll'
 
 const composeInitialPageState = async (store, route) => {
   try {
     await store.dispatch('attribute/list', { // load filter attributes for this specific category
-      filterValues: config.products.defaultFilters, // TODO: assign specific filters/ attribute codes dynamicaly to specific categories
+      filterValues: uniq([...config.products.defaultFilters, ...config.entities.productListWithChildren.includeFields]), // TODO: assign specific filters/ attribute codes dynamicaly to specific categories
```

Comment:
To be on the safe side please add the `config.entities.productListChildren.includeFields` - as from 1.11 @patzick is using only this set of fields in the Category page. The rest is fine. Thanks!

Reply:
@pkarw There already is the `config.entities.productListChildren.includeFields` - do you mean keep only this field and remove this `uniq` function call with `config.products.defaultFilters`?

Reply:
No, sorry I meant `config.entities.productList.includeFields` - doing uniq over `config.entities.productListChildren.includeFields` + `config.entities.productList.includeFields`.

However - I think that Patryk is currently using `config.entities.productListChildren.includeFields` (I doublechecked it) so this comment makes no sense.